src/selector-sizzle.js

```diff
@@ -5,6 +5,8 @@ define( [
 
 jQuery.find = Sizzle;
 jQuery.expr = Sizzle.selectors;
+
+// Deprecated
 jQuery.expr[ ":" ] = jQuery.expr.pseudos;
```

Comment:
I thought it would be odd considering we have selector-native, which doesn't include this.